command/docker.go

```diff
@@ -61,14 +62,14 @@ func (d *DockerManager) ComposeFileUp(composeFilePath string, opts ...pulumi.Res
 
 	return d.runner.Command(
 		d.namer.ResourceName("run", composeFilePath),
-		pulumi.Sprintf("docker-compose -f %s up --detach --wait --timeout 300", remoteComposePath),
-		nil,
-		pulumi.Sprintf("docker-compose -f %s down -t 300", remoteComposePath),
-		nil, true,
+		&CommandArgs{
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Love this !